docs/src/links.jmd

```diff
@@ -0,0 +1,248 @@
+# Numerical considerations when fitting GLMs
+
+Before discussing computational aspects of fitting Generalized Linear Models (GLMs) it is helpful to define the linear model and see which properties are being generalized.
+
+### Properties of linear models
+
+The probability model for a linear model can be written
+\begin{equation}
```

Comment:
AFAICT that's a Weave.jl Markdown file, but I'm not sure what this implies in terms of syntax.